pages/common/ogrmerge.py.md

```diff
@@ -0,0 +1,20 @@
+# ogrmerge.py
+
+> Merge several vector datasets into a single one.
+> More information: <https://gdal.org/programs/ogrmerge.html>.
+
+- Create a GeoPackage with a layer for each input Shapefile:
+
+`ogrmerge.py -f {{GPKG}} -o {{path/to/merged}}.gpkg {{path/to/}}*.shp`
```

Comment:
We use to use `path/to/file1`, `path/to/file2`, ... instead of wildcards :thinking:

Reply:
I see. I used the notation you are suggesting in the examples below. 
With the `*` I just wanted to indicate that it is possible to merge many files. 
But maybe there is a better way to express that. I will revise ...